packages/firestore/test/unit/specs/spec_builder.ts

```diff
@@ -1095,9 +1097,13 @@ export class SpecBuilder {
   private addQueryToActiveTargets(
     targetId: number,
     query: Query,
-    resumeToken?: string,
-    readTime?: TestSnapshotVersion
+    resume?: { resumeToken?: string; readTime?: TestSnapshotVersion },
```

Comment:
Since this "resume" type is being used in at least two places in this file, consider adding an `interface` declaration for it.